psydac/api/feec.py

```diff
@@ -86,51 +86,90 @@ def __init__(self, mapping, *spaces):
     #--------------------------------------------------------------------------
     @property
     def dim(self):
+        """dimension of the physical and logical domains, which are assumed to be the same"""
         return self._dim
 
     @property
     def V0(self):
+        """First space of the De Rham sequence : H1 space"""
         return self._spaces[0]
 
     @property
     def V1(self):
+        """Second space of the De Rham sequence :
+        1d : L2 space
+        2d : either Hdiv or Hcurl space
+        3d : Hcurl space"""
         return self._spaces[1]
 
     @property
     def V2(self):
+        """Third space of the De Rham sequence :
+        2d : L2 space
+        3d : Hdiv space"""
         return self._spaces[2]
 
     @property
     def V3(self):
+        """Fourth space of the De Rham sequence : L2 space in 3d"""
         return self._spaces[3]
 
     @property
     def H1vec(self):
+        """Vector H1 space built as cartesian product of V0 n times with n = dimension of (logical) domain"""
```

Comment:
actually I would specify `Vector-valued H1 space ... `

Reply:
Done in [49ed53e](https://github.com/pyccel/psydac/pull/349/commits/49ed53eec0a2ac1e5117de0e818720027b2a6e36)